openapi.yaml

```diff
@@ -51,6 +54,63 @@ info:
   version: 0.1.4
 openapi: 3.0.2
 paths:
+  /federated/ingest_metadata:
+    post:
+      description: Decrypt payload, process metadata, file secret and send success
```

Comment:
```suggestion
      description: Decrypt payload, process metadata, file secret id and send success
```

Reply:
Changed in 50ac60480d47978cbffb98758dd259f95c18bc3d